hibernate-reactive-core/src/test/java/org/hibernate/reactive/EagerUniqueKeyTest.java

```diff
@@ -56,8 +64,8 @@ public void testMergeDetached(TestContext context) {
 				.thenCompose( i -> getSessionFactory()
 						.withTransaction( session -> session.merge( new Foo( bar ) ) ) )
 				.thenCompose( result -> getSessionFactory()
-						.withTransaction( session -> session.fetch( result.bar )
-						.thenAccept( b -> context.assertEquals( "unique2", b.key ) )
+						.withTransaction( session -> session.fetch( result.getBar() )
+						.thenAccept( b -> context.assertEquals( Bar.class.getName(), b.getClass().getName() ) )
```

Comment:
Did you change the assertion on purpose?

Shouldn't it be
```
.thenAccept( b -> context.assertEquals( "unique2", b.getKey() ) )?
```
?

Reply:
Good point!

Reply:
Done